grails-app/services/org/pih/warehouse/inventory/InventoryService.groovy

```diff
@@ -838,9 +846,9 @@ class InventoryService implements ApplicationContextAware {
      * @param entries
      * @return
      */
-    List getQuantityByBinLocation(List<TransactionEntry> entries, boolean includeOutOfStock) {
+    List<QuantityByBinLocationDto> getQuantityByBinLocation(List<TransactionEntry> entries, boolean includeOutOfStock) {
```

Comment:
I don't like the Dto suffix. It's adds little value and is distracting. I'm also not a fan of the class name, but it is understandable (you're just following the convention of the method name which was poorly named by me). But replicating the method name makes me think that we're going to be creating separate DTOs for every method we refactor which would be a problem.

So let's drop the Dto and come up with a decent name that conveys the data that is being transferred. From what I gather, the granularity is basically the same as the AvailableItem class with a few more properties. The issue there is that we'd need to deal with the "status". 

So we can either make this work with AvailableItem, a similarly named class, or just continue to use the map until we resolve the naming issue.

Reply:
Sure, I have not felt confident naming those, because I have not been entirely sure what they refer to actually.
As I explained at the beginning - it's been difficult to debug those without having the types.
Btw. we already have a dto named `AvailabileItem`, so do you maybe have any other idea, how to name this one?

Reply:
To be honest, I feel like we should somehow distinguish classes that are DTOs. When I joined this project and saw `OutboundStockMovement` I thought that this was a normal database entity, and it took me a while to understand that this exact class is a DTO. Maybe we should align on some exact naming regarding DTOs? Or maybe a good idea would be to place them in a directory called DTO and still use the same class names as we had before?
cc @awalkowiak @drodzewicz

Reply:
Decided to use org.pih.warehouse.inventory.BinLocationItem for now.